sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -5665,6 +6004,61 @@
       "properties": {},
       "lockKey": true,
       "calculateValue": "value = data.offeringIntensity === 'Full Time' ? 'Full-time' : 'Part-time';"
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "isInstitutionBCPublic",
+      "label": "Is institution BC Public",
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "lockKey": true
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "isInstitutionBCPrivate",

```

Comment:
Can we follow the same naming conventions from the designation agreement and programs: `isBCPrivate` and `isBCPublic`?